dedalus/core/basis.py

```diff
@@ -2681,6 +2738,12 @@ def global_grid_colatitude(self, scale):
         theta = self._native_colatitude_grid(scale)
         return reshape_vector(theta, dim=self.dist.dim, axis=self.axis+1)
 
+    def grid_spacing(self, axis, scales=None, r=None):
+        """Global grids spacings."""
+        if scales is None: scales = (1,1)
```

Comment:
`scales` is not used

Reply:
scales is now passed through to local_grid_spacing (https://github.com/evanhanders/dedalus/blob/d3_cfl/dedalus/core/basis.py#L2800); this makes sure the returned array is the right shape for the input scales.